### PR TITLE
V1.7 stable - WinCE compatibility

### PR DESCRIPTION
**Pull request by ondraco**
Added few changes which makes compiling on windows easier (v1.7-stable should be compilable with no problem).

### CODE REVIEW

File: lib/libwebsockets.c [hunk 1]
```diff
@@ -308,6 +308,9 @@ lws_close_free_wsi(struct lws *wsi, enum lws_close_status reason)
 		if (n)
 			lwsl_debug("closing: shutdown ret %d\n", LWS_ERRNO);
 
+// This causes problems with disconnection when the events are half closing connection
+// FD_READ | FD_CLOSE (33)
+#ifndef _WIN32_WCE
```
Comment (JoakimSoderberg) [inline lib/libwebsockets.c]: POSIX + WIN32_WCE ? :)

Comment (ondraco) [inline lib/libwebsockets.c]: Seems so it is just defined as:...

```
#ifdef MBED_OPERATORS
struct sockaddr_in;
#define LWS_POSIX 0
#else
#define LWS_POSIX 1
#endif
```

Comment (lws-team) [inline lib/libwebsockets.c]: No you are right, POSIX means that it's a platform that follows posix semantics for its apis, windows basically does that.

On the other hand mbed3 is a case where the operating system itself is a single global event loop.  It has some posix-sounding apis but they are fundamentally different.

**Comment by lws-team** [comment]
Thanks.  Unless there's a big divergence in some contributed code, I can deal with adapting patches between master and v1.7-stable just as part of normal business.  I just need a patch on one or the other.

If I missed the point and there's something different here, let me know.

**Pull request closed without merging**